src/main/scala/scala/tools/partest/nest/Runner.scala

```diff
@@ -765,7 +765,7 @@ class SuiteRunner(
 
   def banner = {
     val baseDir = fileManager.compilerUnderTest.parent.toString
-    def relativize(path: String) = path.replace(baseDir, "$baseDir").replace(PathSettings.srcDir.toString, "$sourceDir")
+    def relativize(path: String) = path.replace(baseDir, s"$$baseDir").replace(PathSettings.srcDir.toString, "$sourceDir")
```

Comment:
Even without the PR for the constant folding f-interpolator, `f"$$x"` is a constant string.